src/test/cli.build.test.ts

```diff
@@ -170,5 +171,29 @@ describe('Dev Containers CLI', function () {
 			assert.equal(response.imageName[0], image1);
 			assert.equal(response.imageName[1], image2);
 		});
+
+		it('should fail with --push true and --output', async () => {
+			let success = false;
+			const testFolder = `${__dirname}/configs/dockerfile-with-target`;
+			try {
+				await shellExec(`${cli} build --workspace-folder ${testFolder} --output type=oci,dest=output.tar --push true`);
+			} catch (error) {
+				assert.equal(error.error.code, 1, 'Should fail with exit code 1');
+				const res = JSON.parse(error.stdout);
+				assert.equal(res.outcome, 'error');
+				assert.match(res.message, /cannot be used with/);
+			}
+			assert.equal(success, false, 'expect non-successful call');
+		});
+
+		it('file /tmp/output.tar should exist when using --output type=oci,dest=/tmp/output.tar', async () => {
```

Comment:
The `docker` builder doesn't support that option it seems. Could you change the test to first create a builder and remove that builder at the end of the test?

Will you need to do the same for the GitHub action?

Reply:
Also I'm pretty sure the GitHub action environment uses the non-default buildx driver by default but I'll make sure to test that out!

Reply:
I liked your idea of removing the builder at test end rather than leaving it sit. Done.